HHS/Head-Start-TTADP
# [TTHHUB-2651] fix r9 FEI goals

## Description of change

This rewrites history in the sense of changing the text even on approved ARs, changing them to be standard FEI goals. Because this could create some strange "merge" type situations, there's some protective logic to make sure that the transaction won't complete if there's true FEI data that would collide.

## How to test

there's a before_count and after_count to inspect and also you can look at https://ttahub.ohs.acf.hhs.gov/activity-reports/view/38756 to see the terminology updated

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2651


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
